docs/articles/documentation/guides/advanced-guides/authentication.md

```diff
@@ -1,50 +1,37 @@
 ---
 layout: docs
-title: User Roles
-permalink: /documentation/test-api/authentication/user-roles.html
+title: Authentication
+permalink: /documentation/guides/advanced-guides/authentication.html
 ---
-# User Roles
+# Authentication
+
+TestCafe ships with the [user roles](#user-roles) mechanism that covers most authentication types that require user actions on the page.
+
+You can also test websites that use [HTTP Basic](https://en.wikipedia.org/wiki/Basic_access_authentication) and [NTLM](https://en.wikipedia.org/wiki/NT_LAN_Manager) authentication. See [HTTP/NTLM Authentication](#http-authentication) for more information on how to provide the credentials.
+
+## User Roles
 
 Many test scenarios involve the activity of more than one user. TestCafe addresses these scenarios by providing a convenient way
 to isolate authentication test actions and apply them easily whenever you need to switch the user account.
 
 A piece of logic that logs in a particular user is called a *role*. Define a role for each user account participating in your test.
 
-* [Why Use Roles](#why-use-roles)
-* [Create and Apply Roles](#create-and-apply-roles)
-* [Anonymous Role](#anonymous-role)
-* [Role Options](#role-options)
-  * [options.preserveUrl](#optionspreserveurl)
-* [Troubleshooting](#troubleshooting)
-  * [Test Actions Fail After Authentication](#test-actions-fail-after-authentication)
-
-## Why Use Roles
+### Why Use Roles
 
 Unlike other methods used to [extract reusable test logic](../../recipes/extract-reusable-test-code/README.md), roles were specially designed for login operations and provide the following dedicated features:
 
 * **Object-based API.** Authentication data and logic are stored in an object that is easy to pass around and activate when needed.
```

Comment:
Is "pass around and activate" the correct technical terminology? (Just checking for my understanding.)

Reply:
I sometimes come across 'pass around' when talking about variables and objects, but I'm okay with 'pass' instead: 

*Authentication data and logic are stored in an object that is easy to pass and activate when needed.*

Reply:
Okay, thank you